src/test/java/seedu/address/logic/commands/eventcommands/AddPollCommandTest.java

```diff
@@ -33,7 +33,7 @@ public void execute_acceptedAddPoll() {
         //eventBuilder.withOrganiser(user);
         //Event event = eventBuilder.withPoll().build();
```

Comment:
remove the unnecessary comments?

Reply:
would be good to add a test for the NoEventSelectedException in AddPollOptionCommand.java as shown in coveralls as well.

Reply:
Ok wrt the commented out code. But the test for no event selected is there alr no?

Reply:
Ok I realised the error. Will correct in next PR